providers/os/resources/os.lr

```diff
@@ -953,3 +953,29 @@ yum.repo {
   // Indicates if this repository is used as package source
   enabled() bool
 }
+
+// Container Image
+container.image {
+  // Image reference
+  reference string
```

Comment:
I do not understand the difference between `reference` and `name`. Do you have an example?

The comment in the code shows the same value in my understanding.

Reply:
The reference is what you type in docker, for example, when you run a container. So `mondoohq/cnspec:latest`. However, the name is actually the resolved value of that reference, which is `index.docker.io/mondoohq/cnspec:latest`

Reply:
Thanks!